packages/next/src/plugins/plugin.ts

```diff
@@ -36,6 +37,12 @@ export interface NextPluginOptions {
   watchDepsTargetName?: string;
 }
 
+const defaultOptions: NextPluginOptions = {
+  buildTargetName: 'build',
+  devTargetName: 'dev',
+  startTargetName: 'start',
+};
```

Comment:
ommited the deprecated value here since the startTargetName is also in the defaults
serveStaticTargetName?: string;